docs/docusaurus.config.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
   onBrokenMarkdownLinks: 'warn',
   favicon: 'img/favicon.ico',
   organizationName: 'AnitaB.org',
```

Comment:
```suggestion
  organizationName: 'anitab-org',
```

Reply:
Suggested because the organizationName should match with GitHub org name which at present differs

Reply:
Nice catch @vj-codes 🙌🏾